### PR TITLE
feat(UIForm): widget property in uiSchema instead of type

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
**WARNING** : this is for the new implementation that is not available yet
To stick a little more with previous api of uiSchema (and to make @frassinier and TCOMP happy), we should use `widget`property instead of `type`.

**What is the chosen solution to this problem?**
Use `widget`instead of `type` in uiSchema
Before
```
{
      "type": "fieldset",
      "title": "My awesome USER form",
      "items": []
}
```
Now
```
{
      "widget": "fieldset",
      "title": "My awesome USER form",
      "items": []
}
```
**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Comment by talend-glorieux** [comment]
Why did you take the fields title out of the jsonSchema?

File: packages/forms/stories-core/json/core-fieldset.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
       ]
     },
     {
-      "type": "fieldset",
+      "widget": "fieldset",
```
Comment (charlesnguyen) [inline packages/forms/stories-core/json/core-fieldset.json]: line 79:  "type": "textarea" => "widget": "textarea"

**Comment by jsomsanith** [comment]
@talend-glorieux They are in the uiSchema now since this is tied to the form and not the model definition. But using jsfc, they merge the 2 and either we put them in jsonSchema or uiSchema, we will have them

File: packages/forms/stories-core/json/core-fieldset.json (hunk 1 above)
Comment (jsomsanith) [inline packages/forms/stories-core/json/core-fieldset.json]: Changed

**Pull request merged**